### PR TITLE
fix: Final steps

### PR DESCRIPTION
**Pull request by NataliiaKononikhina**
fix: Add hand img

### CODE REVIEW

**Comment by kparzecki** [comment]
image
Please use a non-breaking hyphen here (`&#8209;`).

**Comment by kparzecki** [comment]
image

It would be nice to have some padding below next/prev article links.

**Comment by kparzecki** [comment]
image

Images are overflowing on documentation pages.

**Comment by kparzecki** [comment]
image

I think it would be better to wrap text instead of adding horizontal scroll.

**Comment by kparzecki** [comment]
image

On mobile screen the right part of the last card shadow is not visible in the `DIG INTO THE DOCS` section.

**Comment by kparzecki** [comment]
image

On larger screens the background of the `Build a full-scale global e-commerce site in weeks` section is animated too late making the text invisible.

**Comment by NataliiaKononikhina** [comment]
@kparzecki this PR is about landing page

**Comment by kparzecki** [comment]
image

Content is overflowing when screen width is about 1000px

**Comment by kparzecki** [comment]
image

It would be nice to have some space between logo and first menu element on smaller screens

**Comment by kanstantsin-nekhniadovich** [comment]
Order Api  header is not aligned with other headers.
image

File: user/themes/centra/scss/landing-page/_developers-first-bar.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@
 .developers-first-bar {
   width: 100%;
   overflow: hidden;
-  background-color: #FDB984;
+  background-color: #ffffff;
```
Comment (kanstantsin-nekhniadovich) [inline user/themes/centra/scss/landing-page/_developers-first-bar.scss]: Maybe there is a point to add white color to variables.
It's using quite often.

**Pull request merged**